docs/howto/pull-requests.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Pull Requests
+description: Improve collaboration over data with Pull Requests.
+parent: How-To
+redirect_from: 
+  - /reference/pull_requests.html
+---
+
+# Pull Requests
```

Comment:
Can you please add an RBAC section?

Reply:
AFAIK (maybe @N-o-Z will know better), this feature is currently available to all users.
Plus, if an admin will change the RBAC conf for PRfD, the tab (and "Create PR" button) will still show up for users, but yield errors when trying to act - 
So maybe it's better to leave RBAC out of this doc until it's more mature?

Reply:
Some have it (e.g. morroring) but we're inconsistent. A valid point, we can go ahead without it because it is easy to understand from RBAC page.